chart/templates/worker-deployment.yaml

```diff
@@ -105,14 +129,46 @@ spec:
         {{- if $root.Values.clamav.enabled  }}
         - name: clamav
           image: "{{ $root.Values.clamav.image.value }}:{{ $root.Values.clamav.image.tag }}"
-          imagePullPolicy: {{ $root.Values.image.pullPolicy }}
+          imagePullPolicy: {{ $root.Values.clamav.image.pullPolicy }}
```

Comment:
:+1:

Reply:
This is particularly important for clamAV - because of the frequent changes, I believe this is a prime example of the rarely recommended use of a floating tag (i.e. "latest"). The goal being that we would want clamav to be upgraded to the new version even if no other upgrade to vets-api occurred. This would in basically treat clamav as though it were it's own isolatred deployment. For now though, consistency with existing is the primary name of the game - this change is really just correcting a copy/paste error as the pullpolicy was already in the values yaml, the values yaml reference was simply never updated